openshift/operator-controller.Dockerfile

```diff
@@ -1,14 +1,21 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20 AS builder
 WORKDIR /build
 COPY . .
-RUN make go-build-local
+RUN make go-build-local && \
+    # Build the OLMv1 Test Extension binary.
+    # This is used by openshift/origin to allow us to register the OLMv1 test extension
+    # The binary needs to be added in the component image and OCP image
+    cd openshift/tests-extension && \
+       make build && \
+       mkdir -p /tmp/build && \
+       cp ./bin/olmv1-tests-ext /tmp/build/olmv1-tests-ext && \
+       gzip -f /tmp/build/olmv1-tests-ext
 
 FROM registry.ci.openshift.org/ocp/4.20:base-rhel9
 USER 1001
 COPY --from=builder /build/bin/operator-controller /operator-controller
-COPY openshift/operator-controller/cp-manifests /cp-manifests
+COPY --from=builder /tmp/build/olmv1-tests-ext.gz /usr/bin/olmv1-tests-ext.gz
 COPY openshift/operator-controller/manifests /openshift/manifests
-COPY openshift/operator-controller/manifests-experimental /openshift/manifests-experimental
```

Comment:
@camilamacedo86 I am not sure why you remove this line.  when we add binary into component image, no need to remove it. if there is other reason to remove it, please do it. or else, better to keep it.

Reply:
My main branch had not it at the time.
Yes, we should keep

Reply:
Fixed